src/IO/H5/File.cpp

```diff
@@ -21,7 +22,8 @@
 
 namespace h5 {
 template <AccessType Access_t>
-H5File<Access_t>::H5File(std::string file_name, bool append_to_file)
+H5File<Access_t>::H5File(std::string file_name, bool append_to_file,
+                         std::string input_source)
```

Comment:
const ref unless you end up moving it later.

Reply:
Yeah I'm going to std::move it...clang-tidy caught that as well